Code/GraphMol/Depictor/testDepictor.cpp

```diff
@@ -1539,6 +1541,38 @@ M  END
     TEST_ASSERT(res < 0.);
     TEST_ASSERT(MolToMolBlock(*zeroCoordBenzene) == zeroCoordCTab);
   }
+  {
+    // cyclopentadiene which is already straight should not be biased
+    // towards a 30-degree angle rotate since it has no bonds
+    // whose angle with the X axis is multiple of 60 degrees
+    std::string cpSittingOnHorizontalBondCTab = R"RES(
+  MJ201100                      
+
+  5  5  0  0  0  0  0  0  0  0999 V2000
+   -2.3660    0.3892    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -3.0334   -0.0957    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -2.7785   -0.8803    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -1.9535   -0.8803    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -1.6986   -0.0957    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  1  0  0  0  0
+  1  5  1  0  0  0  0
+  2  3  2  0  0  0  0
+  3  4  1  0  0  0  0
+  4  5  2  0  0  0  0
+M  END
+)RES";
+    std::unique_ptr<RWMol> cpSittingOnHorizontalBond(MolBlockToMol(cpSittingOnHorizontalBondCTab));
```

Comment:
not a big deal, but this would be simpler as:

```suggestion
    auto cpSittingOnHorizontalBond = R"RES(
  MJ201100                      

  5  5  0  0  0  0  0  0  0  0999 V2000
   -2.3660    0.3892    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
   -3.0334   -0.0957    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
   -2.7785   -0.8803    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
   -1.9535   -0.8803    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
   -1.6986   -0.0957    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
  1  2  1  0  0  0  0
  1  5  1  0  0  0  0
  2  3  2  0  0  0  0
  3  4  1  0  0  0  0
  4  5  2  0  0  0  0
M  END
)RES"_ctab;
```

Reply:
Done.